src/power_divergence.jl

```diff
@@ -337,18 +340,18 @@ end
 #convenience functions
 
 #PDT
-function PowerDivergenceTest(x::AbstractVector{T}, y::AbstractVector{T}, levels::Levels{T}; lambda::U=1.0) where {T<:Integer,U<:AbstractFloat}
+function PowerDivergenceTest(x::AbstractVector{T}, y::AbstractVector{T}, levels::Levels{T}; lambda::U=1.0, ddof::Int64=0) where {T<:Integer,U<:AbstractFloat}
```

Comment:
Can you add line breaks when the line goes beyond the 92-char limit? For example:
```suggestion
function PowerDivergenceTest(x::AbstractVector{T}, y::AbstractVector{T}, levels::Levels{T};
                             lambda::U=1.0, ddof::Int64=0) where
                             {T<:Integer,U<:AbstractFloat}
```
(I know the file didn't really follow this rule but let's not make things worse;)

Reply:
I will be happy to fix the line breaks. I will push them later today.

Reply:
I fixed all the long lines in `power_divergence.jl`. It took me a couple of commits because I broke some lines before `where` statements, but I fixed all of them (eventually).